<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1579</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButtonNormal">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходный</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMask">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>50</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Маска</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNoise">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Шумы</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTarget">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>110</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Предмет</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonResult">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>140</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderHDown">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>200</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>179</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelColorDown">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>170</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderSDown">
    <property name="geometry">
     <rect>
      <x>1421</x>
      <y>230</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVDown">
    <property name="geometry">
     <rect>
      <x>1421</x>
      <y>260</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>205</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H нижн</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S нижн</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V нижн</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderSUp">
    <property name="geometry">
     <rect>
      <x>1421</x>
      <y>350</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderHUp">
    <property name="geometry">
     <rect>
      <x>1421</x>
      <y>320</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>179</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>325</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H верх</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S верх</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V верх</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelColorUp">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>290</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVUp">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>380</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderErodeKernel">
    <property name="geometry">
     <rect>
      <x>1421</x>
      <y>170</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>175</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Эрозия</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderDilationKernel">
    <property name="geometry">
     <rect>
      <x>1421</x>
      <y>200</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>205</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дилация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>10</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMinRadius">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>170</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>400</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderP2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>200</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>199</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>175</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Мин. радиус</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>205</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>P2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonNormal</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMask</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNoise</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTarget</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonResult</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSDown</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderVDown</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHDown</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderVUp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderErodeKernel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDilationKernel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMinRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderP2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
